src/evmTest/scala/io/iohk/ethereum/vm/ThrowSpec.scala

```diff
@@ -0,0 +1,20 @@
+package io.iohk.ethereum.vm
+
+import io.iohk.ethereum.vm.utils.EvmTestEnv
+import org.scalatest.{FreeSpec, Matchers}
+
+// scalastyle:off magic.number
+class ThrowSpec extends FreeSpec with Matchers {
+
+  "EVM running Throw contract" - {
+
+    "should handle throwing" in new EvmTestEnv {
+      val (_, contract) = deployContract("Throw")
+
+      val result = contract.justThrow().call()
+
+      result.error shouldBe Some(InvalidOpCode) // TODO: change to invalid jump destination
```

Comment:
I thought #75 was merged already. We need to do that before merging this one.

But this also raises a question: once this gets merged should we enable EVM tests on Circle CI? pinging @mcsherrylabs